core/src/com/unciv/logic/map/mapgenerator/MapRegions.kt

```diff
@@ -495,7 +506,20 @@ class MapRegions (val ruleset: Ruleset){
             }
         }
 
-        // TODO: Strategic Balance Resources
+        // Place Strategic Balance Resources
+        if (tileMap.mapParameters.mapResources == MapResources.strategicBalance) {
+            val candidateTiles = startTile.getTilesInDistanceRange(1..2).shuffled() + startTile.getTilesAtDistance(3).shuffled()
```

Comment:
Why the split between layers 1,2 and letter 3? This is the same as is you would have taken all the layers at once

Reply:
The placement respects the order of the list, so all tiles in ring 1 and 2 will be tried before any in ring 3 are tried.

Reply:
But that's what getTilesInDistance does as well

Reply:
No, getTilesInDistance does not shuffle the tiles. Small difference perhaps, but it might be noticeable if more resources generally spawn at 6' o clock than in other directions.

Reply:
Ah I see, because it the shuffle
Nice